app/src/main/java/com/example/admin/lab_56_57/ActivityForLab57Fragment.java

```diff
@@ -0,0 +1,140 @@
+package com.example.admin.lab_56_57;
+
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.Spinner;
+
+import com.example.admin.lab_56_57.data.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class ActivityForLab57Fragment extends android.support.v4.app.Fragment {
+
+    private static final String LINEAR_LAYOUT = "Linear Layout";
+    private static final String RELATIVE_LAYOUT = "Relative Layout";
+    private AdapterForLinear linearAdapter_;
+    private AdapterForRelative relativeAdapter_;
+    private Spinner menuSpinner_;
+    private ArrayAdapter<String> spinnerAdapter_;
+
+    public enum Mode {
+        LINEAR(LINEAR_LAYOUT),
+        RELATIVE(RELATIVE_LAYOUT);
+
+        private String text_;
+
+        private Mode(String text) {
+            text_ = text;
+        }
+
+        public String getText() {
+            return text_;
+        }
+
+        public static Mode get(String text) {
```

Comment:
We are used to place class's fields in this order:
1. constants (= static or/and final fields)
2. other fields
3. public static methods
4. constructors
5. public methods
6. private methods